serine.galstyan/repo/Lesson_3/exercise_11/exercise_11.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/evn python3
+
+width = float(input("Enter the width(in feet): "))
+length = float(input("Enter the length(in feet): "))
+feets_in_an_acre = 43560
```

Comment:
CSW: Constant value

Reply:
The feets_in_an_acre variable represents 43560 number, as it was indicated to do in the previous comment.